### DISCUSSION TITLE
Federation support

### DISCUSSION THREAD

**Discussion by ToLToL** [minio/minio#12336] [Q&A]
Hi,

The federation support is vital to us because we need a global DNS.
I opened issues / asked question on Slack about federation but it seems like MinIO, along with the console, operator, won't support it anymore.

What are the solutions ?

1. Should I give up MinIO ? 
2. Stick to current version of MinIO and never upgrade ?
3. Implement federation by myself with MinIO operator, coreDNS and etcd ?
4. ...

 I'd like to have your feedback on this.
Thanks

**Comment by ToLToL** [comment]
Some thoughts ?

**Reply by klauspost** [reply]
@ToLToL You are asking for specific and complex features to be implemented. Unless there is a wide benefit for our user base we don't do that from community requests.

If you are interested in direct support for your specific use cases, please write `hello@minio.io` where we can discuss options.
